cg/store/filters/status_case_filters.py

```diff
@@ -200,6 +200,30 @@ def filter_compressible_cases(cases: Query, **kwargs) -> Query:
     return cases.filter(Case.is_compressible)
 
 
+def get_not_received_cases(cases: Query, **kwargs) -> Query:
+    return cases.filter(Sample.received_at == None).distinct()
+
+
+def get_received_cases(cases: Query, **kwargs) -> Query:
+    return cases.filter(Sample.received_at != None).distinct()
```

Comment:
Will this method not return all cases with at least one received sample, rather than all cases with only received samples?

Reply:
True! I guess I need a subquery

Reply:
I do genuinely like them

Reply:
I think this is indeed what it does. Also case entries are unique, I don't think the distinct would do anything here. Right?

Reply:
I guess when you join with Sample, there will be an entry for each sample within the case?